assignments/prototypes.js

```diff
@@ -32,6 +54,20 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(humanoidAttrs) {
+  GameObject.call(this, humanoidAttrs);
```

Comment:
You actually don't need to call GameObject in Humanoid. It inherits from CharacterStats, which inherits from GameObject.